RedRock.Calendar.Modules.Users.Service/PasswordHasher.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Cryptography;
+using System.Text;
+
+namespace RedRock.Calendar.Modules.Users.Service
+{
+    public static class PasswordHasher
```

Comment:
this class contains only static functions, so we call it Helper. Please create a folder called Helpers and rename this file to HashingHelper (make it more generic, because it can be used not only for password hashing, but for hashing in general)